src/navbar.js

```diff
@@ -0,0 +1,25 @@
+const BookSection = document.querySelector('.container');
+const MainTitle = document.querySelector('#list-title');
+const AddSection = document.querySelector('.add-section');
+const Contact = document.querySelector('#contacts');
+
+export function ListDisplay() {
+  BookSection.style.display = 'block';
+  MainTitle.style.display = 'block';
+  AddSection.style.display = 'none';
+  Contact.style.display = 'none';
+}
+
+export function NewBook() {
+  AddSection.style.display = 'block';
+  Contact.style.display = 'none';
+  BookSection.style.display = 'none';
+  MainTitle.style.display = 'none';
+}
+
+export function Getinfo() {
+  Contact.style.display = 'block';
+  AddSection.style.display = 'none';
+  BookSection.style.display = 'none';
+  MainTitle.style.display = 'none';
+}
```

Comment:
- [ ] Great job so far your project look great, :clap: but I think you should use arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
Hey Moses.
Many thanks, I started using ES6 earlier ago, though some best synthases are passing out of my eyes lol, thanks for remembering me to use the arrow function, I wish to write a more professional code in the next few days.
I appreciated